playpen/ansible/roles/dev/files/bashrc

```diff
@@ -76,5 +77,25 @@ psmash() {
     deactivate;
 }
 
+ppopulate() {
+    pulp-admin login -u admin -p admin
+    if [ -d $HOME/devel/pulp_rpm ]; then
+        pulp-admin rpm repo create --repo-id=zoo \
+            --feed=https://repos.fedorapeople.org/repos/pulp/pulp/demo_repos/zoo/
+    fi
```

Comment:
Can you remove the creation of zoo from the vagrant-setup.sh file and have it call ppopulate there instead? In general we don't want to add new stuff to that file but in this case I think it makes sense.